samples/stock/demo/live-candlestick/demo.js

```diff
@@ -0,0 +1,1025 @@
+        375.46,
+        374.82,
+        374.92
+    ],
+    [
+        1317894660000,
+        374.82,
+        375.68,
+        374.64,
+        375.668
+    ],
+    [
+        1317894720000,
+        375.62,
+        376.13,
+        375.46,
+        376.13
+    ],
+    [
+        1317894780000,
+        376.14,
+        376.6,
+        375.89,
+        376.34
+    ],
+    [
+        1317894840000,
+        376.39,
+        376.39,
+        375.55,
+        375.99
+    ],
+    [
+        1317894900000,
+        376,
+        376.28,
+        375.42,
+        376.21
+    ],
+    [
+        1317894960000,
+        376,
+        377.38,
+        375.7,
+        376.591
+    ],
+    [
+        1317895020000,
+        376.59,
+        377.46,
+        376.57,
+        376.9348
+    ],
+    [
+        1317895080000,
+        376.9481,
+        377.749,
+        376.84,
+        377.563
+    ],
+    [
+        1317895140000,
+        377.452,
+        377.65,
+        376.43,
+        376.78
+    ],
+    [
+        1317895200000,
+        376.94,
+        377.01,
+        375.75,
+        375.98
+    ],
+    [
+        1317895260000,
+        376.27,
+        377.29,
+        375.95,
+        376.98
+    ],
+    [
+        1317895320000,
+        376.9962,
+        377.3,
+        376.69,
+        376.71
+    ],
+    [
+        1317895380000,
+        376.75,
+        377.5,
+        376.75,
+        377.41
+    ],
+    [
+        1317895440000,
+        377.26,
+        377.49,
+        376.89,
+        377.368
+    ],
+    [
+        1317895500000,
+        377.345,
+        378,
+        377.17,
+        378
+    ],
+    [
+        1317895560000,
+        377.97,
+        378.3199,
+        377.68,
+        377.97
+    ],
+    [
+        1317895620000,
+        378.01,
+        378.07,
+        377.25,
+        377.37
+    ],
+    [
+        1317895680000,
+        377.37,
+        377.75,
+        377.05,
+        377.12
+    ],
+    [
+        1317895740000,
+        377.16,
+        377.79,
+        377.01,
+        377.4512
+    ]
+];
+
+// Imitate getting point from backend:
+function getNewPoint(i) {
+    const lastPoint = data[data.length - 1];
+
+    // Add new point
+    if (i === 0 || i % 10 === 0) {
+        return [
+            lastPoint[0] + 60000,
+            lastPoint[4],
+            lastPoint[4],
+            lastPoint[4],
+            lastPoint[4]
+        ];
+    }
+    const updatedLastPoint = data[data.length - 1],
+        newClose = Highcharts.correctFloat(
+            lastPoint[4] + Highcharts.correctFloat(Math.random() - 0.5, 2),
+            4
+        );
+
+    // Modify last data point
+    return [
+        updatedLastPoint[0],
+        data[data.length - 2][4],
+        newClose >= updatedLastPoint[2] ? newClose : updatedLastPoint[2],
+        newClose <= updatedLastPoint[3] ? newClose : updatedLastPoint[3],
+        newClose
+    ];
+}
+
+Highcharts.stockChart('container', {
+
+    chart: {
+        height: 800,
```

Comment:
I think the default height is good enough. Then the size of the demo's thumbnail will match other demos. The same in the CSS file

Reply:
I thought about it but I was afraid that the update of the candle won't be visible enough - but I'll change that 👍